daras_ai_v2/realtime_llm_openai.py

```
@@ -205,6 +235,39 @@ def handle_twilio_ws(
+    """Format entry for transcription display"""
+
+    timestamp = datetime.now()
+    content = entry.get("content", "")
+    time_str = format_timestamp(timestamp, conversation_start)
+
+    if entry.get("role") == "user":
+        formatted_content = f"[{time_str}] User: {content}"
+    elif entry.get("role") == "assistant":
+        formatted_content = f"[{time_str}] Assistant: {content}"
+
+    transcription_entry.append(formatted_content)
```

Pylint (3.3.7)

[error] Possibly using variable 'formatted_content' before assignment

(E0606)